astropy/io/votable/tree.py

```diff
@@ -2941,8 +2941,10 @@ def _get_binary_data_stream(self, iterator, config):
         else:
             if not href.startswith(("http", "ftp", "file")):
                 vo_raise(
-                    "The vo package only supports remote data through http, "
-                    + "ftp or file",
+                    (
+                        "The vo package only supports remote data through http, "
+                        "ftp or file"
+                    ),
```

Comment:
I think line continuation would be much better than these extra parenthesis and extra indentation. Why is this needed? I think this will work too:
```suggestion
                    "The vo package only supports remote data through http, "
                    "ftp or file",
```

Reply:
How simple would you say it is to figure out the difference between the following code snippets?
```python
vo_raise(
    "The vo package only supports remote data through http, "
    "ftp or file",
    self._config,
    self._pos,
    NotImplementedError,
)
```
```python
vo_raise(
    "The vo package only supports remote data through http, ",
    "ftp or file",
    self._config,
    self._pos,
    NotImplementedError,
)
```

Reply:
I get your point about confusion with extra args now, but still don't find the extra parentheses on extra lines terribly readable either. Perhaps just put the entire string on one line (and all other args onto another) and protect them from black, after all?

Or try something completely different:
```python
           vo_prot =  ['http', 'https', 'ftp', 'file']
           if href.split(":")[0] not in vo_prot:
                vo_raise(
                    f"The vo package only supports remote data through {vo_prot}",
```
That would also keep bogus protocols like `httpx://...` or `filename:...` from being passed to urllib.

Reply:
> How simple would you say it is to figure out the difference between the following code snippets?

I actually have no problem figuring out differences right away. And probably any unit test would fail too.

Reply:
Done!

Reply:
ping @dhomeier @eerovaher if you approve this change.